targets/csharp-unityv2/source/Shared/Models/SharedModels.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace PlayFab.SharedModels
```

Comment:
this is in preparation for moving models to a shared location between client server & admin...